app/src/main/kotlin/at/bitfire/davdroid/resource/LocalAddressBook.kt

```diff
@@ -85,7 +84,7 @@ open class LocalAddressBook @AssistedInject constructor(
      */
     open val groupMethod: GroupMethod by lazy {
         val manager = AccountManager.get(context)
-        val associatedAccount = manager.getUserData(/* address book account */ account, USER_DATA_COLLECTION_ID)?.toLongOrNull()?.let { collectionId ->
```

Comment:
Now there's `addressBookAccount` in the constructor and as a property. These can be confused and make a difference. Using `addressBookAccount` from the constructor would be bad here because the account could have been renamed and only the `addressBookAccount` property contains the correct account.

So I suggest to rename the constructur argument to `_addressBookAccount` for clarity and add the explanation with renaming in its KDoc.